py3status/modules/volume_status.py

```diff
@@ -68,44 +71,35 @@
 import re
 import shlex
 
-from subprocess import check_output, call
+from subprocess import check_output, call, DEVNULL
 
+from abc import ABCMeta, abstractmethod
+from six import with_metaclass
 
-class Py3status:
-    """
-    """
-    # available configuration parameters
-    button_down = 0
-    button_mute = 0
-    button_up = 0
-    cache_timeout = 10
-    channel = 'Master'
-    device = 'default'
-    format = u'♪: {percentage}%'
-    format_muted = u'♪: muted'
-    threshold_bad = 20
-    threshold_degraded = 50
-    volume_delta = 5
 
-    # compares current volume to the thresholds, returns a color code
-    def _perc_to_color(self, string):
-        try:
-            value = int(string)
-        except ValueError:
-            return self.py3.COLOR_BAD
+class AudioBackend(with_metaclass(ABCMeta)):
```

Comment:
I'd like to second the request to not use `ABCMeta`. I'm completely fine with using classes and inheritance, but instead of using `@abstractmethod`, either `raise NotImplemented` or just ignore it.